apps/system-e2e/src/addons.ts

```diff
@@ -1,28 +1,24 @@
 import { expect, Locator, Page } from '@playwright/test'
 
-const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms))
-
```

Comment:
Removing the sleep now leaves this operation polling the DOM potentially thousands of times per second. This is not a good idea in general and I would suspect will have a very real impact when running in k8s. What is the problem with the current code?

Reply:
Indeed, not a good idea. I was getting `execution context changed` errors, and thought this might be it. It wasn't, but I didn't add it back.

Sleeping is back, defaulting to 100 ms